src/Indexing/Indexing.less

```diff
@@ -0,0 +1,127 @@
+.indexing-page {
+  padding: 30px 20px;
+}
+
+.indexing-page b {
+  font-weight: bold;
+  letter-spacing: 1px;
+  color: black;
+}
+
+.action-panel {
+  display: flex;
+  flex-flow: column;
+  align-items: center;
+  padding: 0;
+  background: var(--g3-color__white);
+  border-radius: 5px;
+  border: 1px solid var(--g3-color__silver);
+  width: 500px;
+  color: var(--g3-color__gray);
+  box-shadow: 1px 1px 8px 0 rgba(0, 0, 0, 0.05);
+  -webkit-box-shadow: 1px 1px 8px 0 rgba(0, 0, 0, 0.05);
+  -moz-box-shadow: 1px 1px 8px 0 rgba(0, 0, 0, 0.05);
+  margin: auto;
+  margin-bottom: 30px;
+}
+
+.action-panel form {
+  text-align: center;
+}
+
+.action-panel-title {
+  border-bottom: 1px solid #f5f5f5;
+  width: 100%;
+  padding: 15px 20px;
+  text-align: center;
+  font-size: 20px;
+  color: black;
+}
+
+.action-panel-body {
+  width: 100%;
+  padding: 20px 20px;
+  text-align: left;
+}
+
+.action-panel-footer {
+  border-top: 1px solid #f5f5f5;
+  width: 100%;
+  padding: 10px 20px;
+  text-align: center;
+}
+
+.indexing-page .popup__foot .popup__right-foot {
+  margin: auto;
+}
+
+.indexing-page .popup__foot {
+  padding: 15px;
+  margin-bottom: 0;
+}
+
+.indexing-page .popup__mask {
+  padding-top: 70px;
+}
+
+.indexing-page .popup__message {
+  padding: 0 30px 18px 30px;
+}
+
+.index-files-popup-text {
+  text-align: center;
+  padding: 0 20px;
+  font-size: 14px;
+}
+
+.index-files-popup-big-icon svg {
```

Comment:
seems to me sth is still not quite right about these icon css. i'm getting a icon not correctly centered in the panel. Looks like the 25*25 icon was placed onto the top left corner of a 80*80 box